doc/scurve-plotting-tools.rst

```diff
@@ -0,0 +1,73 @@
+S-curve plotting tools
+======================
+
+The following tools exist for helping to understand scurve data:
+
+1. :doc:`gemSCurveAnaToolkit.py </man/gemSCurveAnaToolkit>`
+2. :program:`plot_noise_vs_trim.py`
+3. :program:`plot_vfat_and_channel_Scurve.py`
+4. :program:`plot_vfat_summary.py`
+5. :program:`summary_plots.py`
```

Comment:
should probably add `plotSCurveFitResults.py` to this list, my mistake for not doing so earlier.